backend/tests/helpers/UserHelper.test.js

```diff
@@ -0,0 +1,40 @@
+const { expect } = require( 'chai' );
```

Comment:
Either improve these tests or remove them.

Reply:
I've added a test case with an empty array and with only one user.